src/pages/ProductList/components/GoodList/GoodList.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import './GoodList.scss';
+
+const GoodList = () => {
+  return (
+    <div className="goodList">
+      <div className="goodListItem">
+        <div className="image">
+          <img className="data" src="/images/incense.jpeg" alt="data" />
+          <img className="in" src="/images/in.jpeg" alt="data" />
```

Comment:
```suggestion
          <img className="data" src="/images/incense.jpeg" alt="data" />
          <img className="in" src="/images/in.jpeg" alt="data" />
```
* 개발하면서 쓰는 모든 것들이 data,item,value 입니다.
* in은 무엇인지 예상하기 어렵습니다.

data,value,item 으로만 적는게 아닌, 
구체적으로 이 img 태그를 설명할 수 있는 className을 적어주세요.
img파일 이름도 in으로 적어놓으면 예상하기 어렵습니다.
예를 들어, 구름 사진이 담겨있는 이미지 파일인데, cl 라고 쓰는거랑 똑같습니다.

Reply:
반영했습니다!